### PR TITLE
Readibility Improvements

### PR DESCRIPTION
**Pull request by trevrobwhite**
Improvements to the spacing to make it easier to read.  Please note the sidebar needs to the increased by 1 character to stop Service Template Groups falling onto two lines.

### CODE REVIEW

File: app/Config/menu.php [hunk 1]
```diff
@@ -107,18 +107,18 @@
...
 				'itc_systemsettings' => [
 					'url' => ['controller' => 'systemsettings', 'action' => 'index'],
-					'title' => 'Systemsettings',
+					'title' => 'System Settings',
 					'icon' => 'wrench',
 					'parent_controller' => 'Administrators'
 				],
 				'itc_cronjobs' => [
-					'url' => ['controller' => 'cronjobs', 'action' => 'index'],
+					'url' => ['controller' => 'Cron Jobs', 'action' => 'index'],
```
Comment (nook24) [inline app/Config/menu.php]: I @trevrobwhite, can you please revert the controller URL back to cronjobs? :)

Comment (trevrobwhite) [inline app/Config/menu.php]: @nook24 Sorry this is done, a mistake on my part.

File: app/Config/menu.php [hunk 2]
```diff
@@ -107,19 +107,19 @@
...
 					'url' => ['controller' => 'systemsettings', 'action' => 'index'],
-					'title' => 'Systemsettings',
+					'title' => 'System Settings',
 					'icon' => 'wrench',
 					'parent_controller' => 'Administrators'
 				],
 				'itc_cronjobs' => [
-					'url' => ['controller' => 'cronjobs', 'action' => 'index'],
-					'title' => 'Cronjobs',
+					'url' => ['controller' => 'Cronjobs', 'action' => 'index'],
```
Comment (jackdown) [inline app/Config/menu.php]: Controller target must be lowercase

Comment (trevrobwhite) [inline app/Config/menu.php]: @jackdown This has been corrected, sorry.

**Pull request merged**